types/src/front/lib/assistant.ts

```diff
@@ -207,21 +205,6 @@ export const GPT_4O_MINI_MODEL_CONFIG: ModelConfigurationType = {
   toolUseMetaPrompt: OPEN_AI_TOOL_USE_META_PROMPT,
   supportsVision: true,
 };
-export const GPT_3_5_TURBO_MODEL_CONFIG: ModelConfigurationType = {
```

Comment:
this one is ok to remove :+1:

Reply:
I actually need this to implement _getGPT35TurboGlobalAgent (at least I need `modelId` / `providerId` / `description` ) and model is needed in SUPPORTED_MODEL_CONFIGS to support `agent_messages_content_parser.getDelimitersConfiguration`